src/integrationTest/java/integration/tests/DatabaseMetaDataTest.java

```diff
@@ -113,17 +118,51 @@ void shouldReturnTable() throws SQLException {
 		assertNull(result.get(TYPE_NAME).get(0));
 	}
 
+	@ParameterizedTest
+	@CsvSource({
+			// table types
+			",,,,tables;query_history;integration_test,",
+			",,,TABLE;VIEW,tables;query_history;integration_test,",
+			",,,VIEW;TABLE,integration_test;tables;query_history,",
+			",,,TABLE,integration_test,query_history;tables",
+			",,,VIEW,query_history;tables,integration_test",
+
+			// table name pattern
+			",,%quer%,,running_queries;query_history,tables;columns;databases",
+			",,%test,,integration_test,tables;columns;databases",
+
+			// schema name pattern
+			",public,,,integration_test,tables;columns;databases",
+			",information_schema,,,tables;columns,integration_test",
+
+			// Somehow running_queries and query_history appear in table_schema=catalog in information_schema.tables and in table_schema=information_schema
```

Comment:
Let's let a relevant team know, if you haven't already, and add a ticket here so we know where to check if this thing is fixed.